Tools/jhbuild/jhbuild-wrapper

```diff
@@ -76,16 +76,22 @@ def update_jhbuild():
 
 
 def clone_jhbuild():
+    JHBUILD_GIT_URLS = [
+        'https://git.gnome.org/browse/jhbuild',
```

Comment:
You can save a redirect if you use to gitlab.gnome.org directly instead of git.gnome.org. Unfortunately, since it's currently broken, and since my clone uses SSH instead of HTTPS, I don't know the exact URL to give you. GitLab is supposedly recovering so we should be able to check soon....

Reply:
Sounds good like a good idea. Unfortunately git.gnome.org is still down. I'll land this now and change the URL in a follow-up PR.

Reply:
It's back! The URL to use is https://gitlab.gnome.org/GNOME/jhbuild.git